types/event-source-polyfill/tslint.json

```
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-empty-interface": false,
+        "interface-over-type-literal": false
+    }
```

```suggestion
    "extends": "dtslint/dt.json"
```
please add those inline, as per readme

---

ah, yeah, i was reading react's typings, alas this is noted

> (There are some legacy lint configs that have additional contents, but these should not happen in new work.)

Thanks!